lib/lbt/bundle/BundleDefinition.js

```diff
@@ -27,5 +27,13 @@ export const SectionType = {
 	 * Usually used as the last section in a merged module to enforce loading and
 	 * execution of some specific module or modules.
 	 */
-	Require: "require"
+	Require: "require",
+
+	/**
+	 * Dependency cache information that lists modules and their dependencies
+	 * of all types: JS, declarative views/fragments.
+	 * Only the dependencies of the modules are stored as 'depCache' configuration.
```

Comment:
Isn't this already clear with the first sentence? To me this sounds like duplicate information or is confusing.

Reply:
Removed it

Reply:
Maybe this was meant to explain that conditional dependencies are not excluded? if so, I would rather state it like that.

Reply:
> Maybe this was meant to explain that conditional dependencies are not excluded? if so, I would rather state it like that.

Ah yeah, that makes sense 👍🏻

Reply:
The conditional dependencies mentioned by @codeworrior are not internal, as they are reflected in the resources.json (`condRequired`). It would be helpful to mention that conditional dependencies are NOT listed in the depCache information.